### DISCUSSION TITLE
JSON Schema support

### DISCUSSION THREAD

**Discussion by gregsdennis** [AykutSarac/jsoncrack.com#131] [Ideas]
Hello!  Do you have any plans on using JSON Schema in this project?  I'm one of the maintainers of the spec, I have my own implementation, and I'm happy to help as needed.

**Comment by AykutSarac** [comment]
Hello, can you elaborate more your plans with JSON Schema in this app?

**Comment by gregsdennis** [comment]
I think there might be a misunderstanding.  I have no plans on adding such support; I'm merely asking if you use JSON Schema or have any plans to, and I'm offering help if so.

**Comment by btiernay** [comment]
It would amazing to have first class JSON Schema visualization support. Something that I've been using lately is https://github.com/coveooss/json-schema-for-humans ([example](https://coveooss.github.io/json-schema-for-humans/examples/examples_js_default/additional_properties.html#subType1)), but having JS-first tool would be much preferred. Ideally this could be an embeddable component as well.
